### ISSUE TITLE
Difficulty in seeking proposals in meeting notes

### ISSUE THREAD

**Issue by oyilmaztekin** [tc39/proposals#188]
Hi Folks 🖐️,

I was searching for proposals in meeting notes for my last improvement about "last presented columns". It is really hard to find anything in these notes. Do you have any plans for this? For this like, building a website or implementing these notes in an existing website would be a better solution. It's really painful this way and it's getting more complex each addition. Therefore, it can be implemented in the future.    

I talked to some UX Designers about this repo, they will share their opinions soon. Besides, maybe we can ask them what we can improve on the beta website.

**Comment by littledan** [comment]
@oyilmaztekin Hi, thanks for your comments. What do you think about working on meeting note presentation in the draft TC39 website, https://github.com/tc39/beta , rather than this repository? (FWIW, for me, the great work that @leobalter and @rwaldron did on the toc.md and summary.md in notes directories was a huge help in looking at past discussions of proposals.)

**Comment by rwaldron** [comment]
@oyilmaztekin I would love to add search to the meeting notes site, but it will be a non-trivial task to do so. Presently, I search the notes using google, with this syntax: `site:tc39.github.io/tc39-notes bigint`

**Comment by oyilmaztekin** [comment]
Let me examine the beta website and check what we can do to improve the usability of this meeting notes. It would be a very useful feature but as @rwaldron said it would be a non-trivial task. I hope we can find a server-side or client-side solution for this. Considering the priorities, I can return this issue after #161 and #140 completed. Is that ok @littledan

**Comment by littledan** [comment]
Sounds good to me (but note, those issues may be more difficult to find consensus on a change, whereas everyone would love search :). Cc @codehag

**Comment by mathiasbynens** [comment]
Another issue I hit often is that GitHub doesn’t support search in forked repositories. 

This makes it impossible to search https://github.com/tc39/tc39-notes because it’s a fork of https://github.com/rwaldron/tc39-notes. Why are there two repositories (and two hosted versions of the notes)?

**Comment by ljharb** [comment]
Now that https://github.com/tc39/notes is in the org, searching should work fine.

I'm going to close this, but if there's anything left that's actionable in this repo I'm happy to reopen it.
